target/linux/ixp4xx/config-6.6

```diff
@@ -85,6 +85,7 @@ CONFIG_FORCE_PCI=y
 CONFIG_FS_IOMAP=y
 CONFIG_FS_MBCACHE=y
 CONFIG_FUNCTION_ALIGNMENT=0
+# CONFIG_FSL_DPAA2_SWITCH is not set
```

Comment:
Why is this change needed? Why is it only needed for these two targets?

Reply:
Maybe this was related to older kernel versions or something else have changed in master. But anyway seems to work without it.